docs/getting_started/requirements.md

```diff
@@ -480,7 +480,7 @@ Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached,
 
     For example:
 
-    - Platform.sh provides Redis support for versions 3.2, 4.0 and 5.0. Ibexa DXP supports Redis version 4.0 or higher, and recommends 5.0. As a result, Redis is supported on Ibexa Cloud in versions 4.0 and 5.0, but 5.0 is recommended.
+    - Platform.sh provides Redis support for versions 6.2, 7.0 and 7.2. Ibexa DXP supports Redis version 4.0 or higher, and recommends 5.0. As a result, Redis is supported on Ibexa Cloud in versions 4.0 and 5.0, but 5.0 is recommended.
```

Comment:
I find this example a bit confusing - we're recommending to use a version that Platform.sh does not support? It doesn't look right 🤔